pkg/asyncdr/asyncdr.go

```diff
@@ -97,6 +97,119 @@ func CreateMigration(
 	return mig, err
 }
 
+func CreateMigrationSchedule(
+	name string,
+	namespace string,
+	clusterPair string,
+	migrationNamespace string,
+	includeVolumes *bool,
+	includeResources *bool,
+	startApplications *bool,
+	sp string,
+	suspend *bool,
+	autoSuspend bool,
```

Comment:
Need to add all the parameters for migration schedule: https://github.com/libopenstorage/stork/blob/7ee546008cb1c09acf7d0cb9ad0d1a223486cd39/pkg/apis/stork/v1alpha1/migration.go#L15

That way we can create all types of migration schedules possible, if required.

Reply:
Added